sources/packages/api/src/route-controllers/user/model/user.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { IsNotEmpty } from "class-validator";
```

Comment:
Why do we need this object? Are we receiving this data from the client?
I believe that this data should come from the token, right?